scripts/netContain/scripts/contivNet.sh

```diff
@@ -16,6 +16,8 @@ cleanup=false
 cstore_param=""
 vtep_ip_param=""
 vlan_if_param=""
+control_url=":9999"
```

Comment:
do you need to pass default control_url/listen_url ?
Isn't that taken care by netmaster if not specified ?